### PR TITLE
Minor UI changes and graceful exit

### PR DESCRIPTION
**Pull request by MRZA-MRZA**
Now Kaidan disconnects on exit.
Also added Flickable to make roster scrollable (drag to scroll).

### CODE REVIEW

File: app/source/Kaidan.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@
 Kaidan::Kaidan(NetworkFactories* networkFactories, QObject *parent) :
     rosterController_(new RosterController()), QObject(parent)
 {
-    client = new Swift::Client("jid@...", "pass", networkFactories);
+	client = new Swift::Client("jid@...", "pass", networkFactories);
```
Comment (geobra) [inline app/source/Kaidan.cpp]: fix indent

File: app/source/main.cpp [hunk 2]
```diff
@@ -22,12 +22,14 @@ int main(int argc, char *argv[])
...
+	QQmlApplicationEngine engine;
+	engine.rootContext()->setContextProperty("kaidan", &kaidan);
 
-    view.setSource(QUrl("qrc:/main.qml"));
-    view.show();
+	engine.load(QUrl("qrc:/main.qml"));
+	QObject *topLevel = engine.rootObjects().value(0);
+	QQuickWindow *window = qobject_cast<QQuickWindow*>(topLevel);
 
+	QObject::connect(topLevel, SIGNAL(mainQuit()), &kaidan, SLOT(mainQuit()));
```
Comment (geobra) [inline app/source/main.cpp]: is connect needed here? Call a slot methode or use Q_IMVOCABLE as described here:
http://doc.qt.io/qt-5/qtqml-cppintegration-exposecppattributes.html

Comment (MRZA-MRZA) [inline app/source/main.cpp]: I have fixed it.

File: app/source/Kaidan.cpp (hunk 1 above)
Comment (MRZA-MRZA) [inline app/source/Kaidan.cpp]: According to https://github.com/KaidanIM/Kaidan/issues/17 the majority voted for tabs

**Comment by LNJ2** [comment]
Here are several things mixed up ... but thats ok for this time, next time please multiple PRs for every feature/clean up/minor edit. @MRZA-MRZA 

But I also approve: :+1:

**Pull request merged**